### PR TITLE
[W5.11][W12-2]Lim Zhi Hui Alden 

### PR DESCRIPTION
**Pull request by E0201942**
Password added. Right password needs to be inputted to access the address book functions.

### CODE REVIEW

**Comment by auskure** [comment]
This could possibly allow our users to feel more secure when using our program.  However, in order to make it fully secure, we might have to implement more underlying features.  E.g. we possibly have to encrypt the saved password.  Overall, good job!

**Comment by ABearPerson** [comment]
This is similar to what i have done!

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -38,6 +38,11 @@ What's different from AddressBook-Level1:
 * Support for marking a contact detail as 'private' (`pa/`) (`pe/`) (`pp/`)
 * View details of a person (`view` : shows non-private details), (`viewall` : shows all details)
 
+
+== Accessing the address book
+Password: default
```
Comment (okkhoy) [inline docs/UserGuide.adoc:43]: 😂 this is very secure!!

File: src/seedu/addressbook/Main.java [hunk 2]
```diff
@@ -70,6 +72,17 @@ private void start(String[] launchArgs) {
              */
             throw new RuntimeException(e);
         }
+    }
+    private void passwordEntry(){
```
Comment (okkhoy) [inline src/seedu/addressbook/Main.java:76]: Add some vertical spacing for better readability

**Review by okkhoy — COMMENTED** [review]
You need to update the userguide informing the user of the changes.

**Comment by okkhoy** [comment]
@E0201942 
Some comments added. Please take note and close the pull request.

**Pull request closed without merging**